tests/integration/src/test/java/org/mule/test/integration/domain/http/HttpSharePortSamePathTestCase.java

```diff
@@ -52,7 +51,7 @@ public HttpSharePortSamePathTestCase(String domainConfig, String appConfig, Clas
     {
         return Arrays.asList(new Object[][] {
                 {"domain/http/transport/http-shared-connector.xml", "domain/http/transport/http-hello-mule-app.xml", ConnectorException.class},
-                {"domain/http/http-shared-listener-config.xml", "domain/http/http-hello-mule-app.xml", InitialisationException.class}});
+                {"domain/http/http-shared-listener-config.xml", "domain/http/http-hello-mule-app.xml", MuleRuntimeException.class}});
```

Comment:
Don't remember which change affected this, but I think is better a InitilialisationException than a plain RuntimeMuleException